composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php":             ">=5.3.2",
         "ext-curl":        "*",
-        "guzzle/guzzle":   "~3.7"
+        "guzzlehttp/guzzle": "~3.7 || ~4.0 || ~5.0 || ~6.0"
```

Comment:
4.x support is not very painful AFAIK.

For 3.x, I'm stuck actually: https://github.com/KnpLabs/php-github-api/pull/317#discussion_r41097885